python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -314,8 +314,12 @@ def processResults(self, coaddExposure, dataRef):
                 exposure=coaddExposure,
                 exposureId=expId
             )
-            psfResults = self.measurePsf.run(coaddExposure, coaddSources, expId=expId)
-            coaddExposure.setPsf(psfResults.psf)
+            try:
+                psfResults = self.measurePsf.run(coaddExposure, coaddSources, expId=expId)
+            except RuntimeError as e:
+                self.log.warn("Unable to calculate PSF, using default coadd PSF: %s" % e)
```

Comment:
Just to double check, will things be able to proceed if `coaddExposure.setPsf` is never called? In other words, please confirm that is it *not* possible for `coaddExposure` to be missing a PSF entirely.

Reply:
Yes, the PSF gets set earlier. So, if `self.config.doCalculatePsf=True` it over-writes the existing PSF.